### ISSUE TITLE
Navigate to hashtag

### ISSUE THREAD

**Issue by schmidt-fu** [33mhz/robin#22]
When I open a post and tap on a hashtag (highlighted like a hyperlink), nothing happens.

It should open a stream of that hashtag.

If not implemented, it could still open a search for that hashtag, ordered by id, so to simulate said stream.

**Comment by 33mhz** [comment]
Yep, it doesn't do anything. `in/lib/view/spannable/HashTagClickableSpan.java` has nothing in it, and no views.
